docs/source/algorithms/HB3AAdjustSampleNorm-v1.rst

```diff
@@ -0,0 +1,23 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Takes in DEMAND detector scan data from files or workspaces and converts them to Q-space, providing an optional
```

Comment:
The `summary` method of the algorithm is already put in the docs as part of the `.. algorithm::` directive above. You don't need to repeat it here.